lib/hanami/cli/generators/gem/app/app_layout.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
Could we drop this line? Seems quite outmoded now. [See here](https://stackoverflow.com/a/6771584).

Reply:
@timriley Cargo-culting at its finest. Sorry about that.
Removed: https://github.com/hanami/cli/pull/99/commits/abbb927ae1dbc2aaede98d78adf43910d86aebd3